### DISCUSSION TITLE
图片移动到了下一个section

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#106] [Q&A]
你好，

`\begin{figure}
  \centering
  \includegraphics[width=0.1\textwidth]{whulogo.pdf}
  \caption{插图示例2}
  \label{fig:whu2}
\end{figure}
\section{公式的使用}
在文中引用公式可以这么写：\(a^2 + b^2 = c^2\)。这是勾股定理，它还可以表示为 \(c = \sqrt{a^2 + b^2}\)。还可以让公式单独一段并且加上编号：`
图片应该在\section{公式的使用}之前，但是实际情况是：
在\section{公式的使用}之后了？这怎么回事呢？
![Screenshot 2022-04-09 122632](https://user-images.githubusercontent.com/45897471/162556240-d0b6694c-5f25-4180-9163-352507668a85.jpg)

**Comment by tanukihee** [comment]
读 lshort 了解浮动体

**Reply by earthmoon** [reply]
> 读 lshort 了解浮动体

明白啦。我觉啊，如果在同一个section里面，那么置于页面的顶部是合适的，如果不是同一个section，就要准确放置图片。有办法实现这个功能吗？

**Reply by xkwxdyy** [reply]
没必要这么折腾

**Reply by earthmoon** [reply]
> 没必要这么折腾

哈哈

**Reply by tanukihee** [reply]
> > 读 lshort 了解浮动体
> 
> 明白啦。我觉啊，如果在同一个section里面，那么置于页面的顶部是合适的，如果不是同一个section，就要准确放置图片。有办法实现这个功能吗？

可以，但 LaTeX 认为不合适

如果你对排版有近乎变态的执着，而且并不太会使用 LaTeX 的话，可能一些专业的排版软件，比如 InDesign 或者方正飞腾更适合你

**Reply by earthmoon** [reply]
慢慢学Latex吧，Indesign引用文献都是难题。科研论文还是latex最好

**Reply by tanukihee** [reply]
选择一个工具就要接受它的所有方便与不便

**Reply by earthmoon** [reply]
> 

是的。

**Comment by tanukihee** [comment]
建议在使用 whu-thesis 之前，先了解一下 LaTeX 的基本概念与使用方法

**Reply by earthmoon** [reply]
> 建议在使用 whu-thesis 之前，先了解一下 LaTeX 的基本概念与使用方法

我找了Ishort的图片P32页，想不到是在浮动体里面。

**Reply by tanukihee** [reply]
建议把 lshort 与 whu-thesis 的文档先完整读两三遍

**Reply by earthmoon** [reply]
> 建议把 lshort 与 whu-thesis 的文档先完整读两三遍

好，那就读3遍。
